css/css-display/display-contents-accessibility.html

```diff
@@ -0,0 +1,321 @@
+        <li>x</li>
+        <li style="display: contents;" data-expectedrole="listitem" data-testname="li with display:contents, as child of ol, has listitem role" class="ex">y</li>
+    </ol>
+
+    <div role="list" style="display: contents;" data-expectedrole="list" data-testname="div with list role and display:contents has list role" class="ex">
+        <div role="listitem" data-expectedrole="listitem" data-testname="div with listitem role, as child of div with display:contents, has listitem role" class="ex">x</div>
+        <div>y</div>
+    </div>
+
+    <div role="list" style="display: contents;" data-expectedrole="list" data-testname="div with list role and display:contents has list role (child div with listitem role has display:contents)" class="ex">
+        <div>x</div>
+        <div role="listitem" style="display: contents;" data-expectedrole="listitem" data-testname="div with listitem role (as child of div with list role), both with display:contents, has listitem role" class="ex">y</div>
+    </div>
+
+    <div role="list">
+        <div>x</div>
+        <div role="listitem" style="display: contents;" data-expectedrole="listitem" data-testname="div with listitem role with display:contents, as child of div with list role, has listitem role" class="ex">y</div>
+    </div>
+
+    <!-- Lists: description lists -->
+    <dl>
+        <dt style="display: contents;" data-expectedrole="term" data-testname="dt with display:contents, as child of dl, has term role" class="ex">q</dt>
+        <dd style="display: contents;" data-expectedrole="definition" data-testname="dd with display:contents, as child of dl, has definition role" class="ex">r</dd>
+        <dt>x</dt>
+        <dd>y</dd>
+    </dl>
+
+    <dl style="display: contents;">
+        <dt style="display: contents;" data-expectedrole="term" data-testname="dt with display:contents, as child of dl with display:contents, has term role" class="ex">q</dt>
+        <dd style="display: contents;" data-expectedrole="definition" data-testname="dd with display:contents, as child of dl with display:contents, has definition role" class="ex">r</dd>
+        <dt>x</dt>
+        <dd>y</dd>
+    </dl>
+
+    <!-- Tables (including grids/treegrids) -->
+    <table style="display: contents;" data-expectedrole="table" data-testname="table with display:contents has table role" class="ex">
+        <thead>
+          <tr style="display: contents;" data-expectedrole="row" data-testname="tr in table with display:contents has row role" class="ex">
+            <th style="display: contents;" data-expectedrole="columnheader" data-testname="td within tr both with display:contents, in table with display:contents, has columnheader role" class="ex">x</th>
+            <th>x</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td style="display: contents;" data-expectedrole="cell" data-testname="td within tr in table, all with display:contents, has cell role" class="ex">x</td>
```

Comment:
See above comments about `cell`/`gridcell` tests possibly needing the containing table to have an explicit `role=table`... this one looks like a legitimate Chromium failure to me, but it's possibly a leftover implementation detail that the `grid`/`gridcell` roles (from ARIA 1.0) predate the `table`/`cell` roles (from ARIA 1.1 or 1.2).

In any case, I think this particularly Chromium failure is unlikely to be caused by any `display:contents` logic.